packages/sdk-manager/src/SdkManagerContextProvider.ts

```diff
@@ -0,0 +1,100 @@
+import { Environment } from '@buildwithsygma/sygma-sdk-core';
+import { consume, createContext, provide } from '@lit/context';
+import { LitElement, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import {
+  WalletManagerContext,
+  WalletManagerController
+} from '@buildwithsygma/sygmaprotocol-wallet-manager';
+import { SdkManagerState } from './types';
+import { SdkManager } from './SdkManager';
+
+export const SdkManagerContext = createContext<SdkManagerState | undefined>(
+  'sdk-context'
+);
+
+/**
+ * @name SdkManagerContextProvider
+ * @description This component is responsible for providing the SdkManagerController as a context to all its children.
+ *
+ * @example
+ * For you to consume the context objects, you need to wrap up your component with the sdk-manager-context on your render method.
+ *
+ * <wallet-manager-context>
+ *   <sdk-manager-context-provider>
+ *     <your-component></your-component>
+ *   </sdk-manager-context-provider>
+ * </wallet-manager-context>
+ */
+
+@customElement('sdk-manager-context-provider')
+export class SdkManagerContextProvider extends LitElement {
+  @consume({ context: WalletManagerContext, subscribe: true })
+  @state()
+  walletManager?: WalletManagerController;
+
+  @provide({ context: SdkManagerContext })
+  @state()
+  sdkManager?: SdkManager;
```

Comment:
cc @wainola 
when refactoring, it'd be good that both managers follow same naming convention